src/AppHeader.tsx

```diff
@@ -32,10 +43,11 @@ export const AppHeader: React.FC = () => {
           <LanguageDropdown />
         </div>
         <div className="appheader__moltincartcontainer">
-          <span
-            className="moltin-cart-button"
-            key={id}
-          ></span>
+          <button className="epbtn --bordered" onClick={handleCartModal}>
```

Comment:
Let's show the cart icon button in mobile like we previously did, to save on space:
![image](https://user-images.githubusercontent.com/39800563/92153217-e8642580-edf1-11ea-924b-093b4766092f.png)